rllib/agents/dqn/simple_q.py

```diff
@@ -112,35 +112,59 @@
 # yapf: enable
 
 
-def get_policy_class(config: TrainerConfigDict) -> Optional[Type[Policy]]:
-    """Policy class picker function. Class is chosen based on DL-framework.
+# TODO: Move into SimpleQ once all OffPolicyTrainers have been converted to
```

Comment:
still applies?
if these are the last remaining classes, why don't move this and execution_plan() below into SimpleQ class?

Reply:
Good catch, this PR should deprecate `OffPolicyTrainer` entirely.